src/subdomains/generic/kyc/services/document-storage.service.ts

```diff
@@ -42,6 +36,21 @@ export class DocumentStorageService {
   }
 
   // --- HELPER METHODS --- //
+  private blobToKycFile(blobs: Blob[]): KycFile[] {
+    return blobs.map((b) => {
+      const [_, type, name] = this.fromFileId(b.name);
+      return {
+        type,
+        name,
+        url: b.url,
+        contentType: b.contentType as KycContentType,
+        created: b.created,
+        updated: b.updated,
+        metadata: b.metadata,
+      };
+    });
+  }
```

Comment:
Maybe it would be easier to have a method `private listFilesByPrefix(prefix: string): Promise<KycFile[]>` and call it from both methods?